README.md

```diff
@@ -27,7 +22,8 @@ sudo python setup.py install
 Or install from pip
 
 ```
-pip install pynuodb
+sudo pip install pynuodb
+sudo pip install --upgrade pynuodb
```

Comment:
I don't think this is quite right. Looking at the documentation for `pip install` it says
```
  -U, --upgrade               Upgrade all specified packages to the newest available version. This process is
                              recursive regardless of whether a dependency is already satisfied.
```

So perhaps something like

if you are installing for the first time
... (without --upgrade)

if you would like to upgrade your installation to the latest version
... (with --upgrade)